.github/workflows/main.yml

```diff
@@ -43,17 +43,27 @@ jobs:
         with:
           failure-threshold: error
 
-  docker-publish:
-    if: github.ref == 'refs/heads/main'
+  tag-new-version:
+    # This uses `conventional commits` to generate tags.  A full list
+    # of valid prefixes is here:
+    # https://github.com/commitizen/conventional-commit-types/blob/master/index.json)
```

Comment:
Is the trailing parenthesis intentional?

Reply:
No, copypasta fail!